lib/statsd/instrument.rb

```diff
@@ -344,9 +344,9 @@ def statsd_remove_distribution(method, name)
       remove_from_method(method, name, :distribution)
     end
 
-    VoidClass = Class.new(BasicObject)
+    VoidClass = Class.new
```

Comment:
Why not just `Object.new?`

Reply:
This gives you a more targeted error message ("`foo` is not a method on `VoidClass`" rather than "`foo` is not a method on `Object`").